api/researches/views.py

```diff
@@ -272,12 +273,18 @@ def researches_update(request):
         spec_pk = request_data.get("speciality", -1)
         speciality = Speciality.objects.filter(pk=spec_pk).first()
         direction_current_form = request_data.get("direction_current_form", 0)
+        result_current_form = request_data.get("result_current_form", 0)
         direction_current_params = request_data.get("direction_current_params", -1)
         researche_direction_current_params = None
         if int(direction_current_params) > -1:
             researche_direction_current_params = DResearches.objects.get(pk=int(direction_current_params))
         if not direction_current_form:
             direction_current_form = 0
+        own_form_result = True
+        if not result_current_form:
+            result_current_form = 0
+        if result_current_form == 0:
+            own_form_result = False
```

Comment:
Зачем так сложно))
Можно ведь просто вместо этих строк – это
```suggestion
            result_current_form = result_current_form or 0
            own_form_result = result_current_form == 0
```

Reply:
Две более понятные вместо четырех